src/pg/sql/getArgSql.js

```diff
@@ -73,10 +73,20 @@ export default function getArgSql(
     getJsonBuildTrusted({ $order: sql`${JSON.stringify($order)}::jsonb` });
 
   const cursorKey = getJsonBuildTrusted({
-    $cursor: sql`jsonb_build_array(${join(groupCols || orderCols)})`,
+    $cursor:
+      $group === true
+        ? sql`jsonb_build_array('')`
+        : sql`jsonb_build_array(${join(groupCols || orderCols)})`,
   });
 
-  key = sql`(${join([key, orderKey, cursorKey].filter(Boolean), ` || `)})`;
+  const groupKey =
+    Array.isArray($group) &&
```

Comment:
I believe this should be `$group &&`? I would expect that the result key must have `$group: true` if the query key had `$group: true`.

Right now it is working because getAggMeta is adding it. But we can remove this logic from getAggMeta as we are doing it here.